projects/vdk-core/src/vdk/internal/core/errors.py

```diff
@@ -410,28 +454,15 @@ def find_whom_to_blame_from_exception(exception: Exception) -> ResolvableBy:
         )  # TODO find out if this is a local or platform deployment and fix this line.
     if issubclass(type(exception), PlatformServiceError):
         return ResolvableBy.PLATFORM_ERROR
-
     return ResolvableBy.PLATFORM_ERROR
 
 
-MSG_CONSEQUENCE_DELEGATING_TO_CALLER__LIKELY_EXECUTION_FAILURE = (
-    "I'm rethrowing this exception to my caller to process."
-    "Most likely this will result in failure of current Data Job."
-)
-MSG_CONSEQUENCE_TERMINATING_APP = (
-    "The provided Data Job will not be executed. Terminating application."
-)
-MSG_COUNTERMEASURE_FIX_PARENT_EXCEPTION = (
-    "See contents of the exception and fix the problem that causes it."
-)
-
-
-def get_exception_message(exception: Exception) -> str:
+def _get_exception_message(exception: Exception) -> str:
```

Comment:
> The only difference is that I renamed _get_eception_message to get_exception_message as not to break backwords compatability.
Didn't get your intention on access modifier

Out of curiosity, what was the usage of `get_exception_message`? And shouldn't we update the usage instead, so that we improve the interface surface in early stages while possible, or is it late already?